homework_2/task2.js

```diff
@@ -0,0 +1,24 @@
+const checkIfStart = str => {
+
+    if (str.length < 2) {
+        return false;
+    }
+    
+    let newString = str.substring(0, 2);
+    
+    if (
+        typeof newString === "string" &&
```

Comment:
method substring always returns the type of string.
Try to check the type of str in the 3rd line